var/spack/repos/builtin/packages/openradioss/package.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import inspect
+import os
+from shutil import move
+
+from spack.package import *
+
+
+class Openradioss(CMakePackage):
+    """Altair Radioss is an industry-proven analysis solution that helps users evaluate and
+    optimize product performance for highly nonlinear problems under dynamic loadings. For more
+    than 30 years, organizations have used Altair Radioss to streamline and optimize the digital
+    design process, replace costly physical tests with quick and efficient simulation, and speed
+    up design optimization iterations – all so users and organizations can improve product quality,
+    reduce costs, and shorten development cycles."""
+
+    homepage = "https://www.openradioss.org/"
+    url = "https://github.com/OpenRadioss/OpenRadioss/archive/refs/tags/latest-20220914.tar.gz"
+
+    version("20220916", sha256="b6f24127ca96d92462f302fecee7b71b33dd85cffb28fd27c9e60cbe26bc8695")
+    version("20220914", sha256="7764c3bbaaee63d78938ea632cde2b90fa56c9ad954aca184b392ae2b76dae16")
```

Comment:
Since it appears the project doesn't retain old versions, perhaps it would be better to rely on the `main` branch (see https://spack.readthedocs.io/en/latest/packaging_guide.html#git).

Reply:
I'd love to add the main branch, but this requires that the project is pulled with `git lfs` enabled. There is no direct `lfs` support in any pf the fetch strategies and I have not been able to make this work with the provided hooks.